app/models/manageiq/providers/base_manager.rb

```diff
@@ -29,6 +29,10 @@ def http_proxy_uri
       self.class.http_proxy_uri
     end
 
+    def console_url
+      raise MiqException::Error, _("Console not supported")
```

Comment:
Prefer this base method raises `NotImplementedError` since this should never be called (`.supports?(:console)` must be checked before calling it)
```suggestion
      raise NotImplementedError,, _("console_url must be implemented in a subclass")
```

Reply:
Thanks Adam. Valid point.